packages/server/lib/html/get-local-storage.html

```diff
@@ -0,0 +1,27 @@
+<html>
```

Comment:
Any way we can pull the server changes into develop now?

Reply:
We could, though I'm not sure we gain much since this isn't actually fixing an issue in develop and there would be a slight disconnect from the issue since it deals with snapshots. 🤷

Reply:
It would be less to review when v8-snapshots is merged into develop

Reply:
If the PR is changed to develop, we wouldn't be able to verify the change is actually correct since snapshots aren't used in develop.

Reply:
wouldn't you be able to verify once the latest develop changes are merged in?

Reply:
Sure, but that is after the fact. You would be trusting me that this fix is correct prior to actually being able to verify the change. Maybe that is okay since we know html files aren't snapshotted but I don't think there's much to gain going to develop first.